content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -564,12 +564,15 @@ spec:
     # the Webhook strategy instructs the API server to call an external webhook for any conversion between custom resources.
     strategy: Webhook
     # webhookClientConfig is required when strategy is `Webhook` and it configures the webhook endpoint to be called by API server.
+    conversionReviewVersions: ["v1", "v1beta1"]
+    # Ensuring compatibility with v1 and v1beta1
```

Comment:
Why these changes?
The changes are incorrect.
Line 566 should appear before line 569, right?

Reply:
Thank you for pointing this out, @tengqm! You're correct that the changes are misplaced. Line 566 should indeed appear before line 569 to ensure the proper structure and flow. I'll go ahead and make that correction.